### PR TITLE
release git-lfs v2.0.0

### PR DESCRIPTION
**Pull request by technoweenie**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,132 @@
...
+
+* ensure you're in the correct directory when installing #1793 (@technoweenie)
+* locking: make API requests relative to repository, not root #1818 (@ttaylorr)
+* Teach 'track' about CRLF #1914 (@technoweenie)
+* Teach 'track' how to handle empty lines in .gitattributes #1921 (@technoweenie)
+* Closing stdout pipe before function return #1861 (@monitorjbl)
+* Custom transfer terminate #1847 (@sinbad)
+* Fix Install in root problems #1727 (@technoweenie)
+* cat-file batch: read all of the bytes #1680 (@technoweenie)
+* Fixed file paths on cygwin. #1820 (@creste)
```
Comment (ttaylorr) [inline CHANGELOG.md]: #1965 also addresses this 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,132 @@
...
+
+### Misc
```
Comment (ttaylorr) [inline CHANGELOG.md]: A few more misc changes:

https://github.com/git-lfs/git-lfs/pull/1954: treat empty pointers as empty files
https://github.com/git-lfs/git-lfs/pull/1952: build on Go 1.8

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,132 @@
...
+* Implement local lock cache, support querying it #1760 (@sinbad)
+* support for client certificates pt 2 #1893 (@technoweenie)
+* Fix clash between progress meter and credential helper #1886 (@technoweenie)
+* Teach uninstall cmd about --local and --system #1887 (@technoweenie)
+* Add `--skip-repo` option to `git lfs install` & use in tests #1868 (@sinbad)
+* commands: convert push, pre-push to use async gitscanner #1812 (@ttaylorr)
+* tq: prioritize transferring retries before new items #1758 (@ttaylorr)
+* Fast walk no channels #1710 (@technoweenie)
+
+### Bugs
```
Comment (ttaylorr) [inline CHANGELOG.md]: A few other bugs:

https://github.com/git-lfs/git-lfs/pull/1962: fixed `GIT_SSH_COMMAND` with spaces
https://github.com/git-lfs/git-lfs/pull/1958 & https://github.com/git-lfs/git-lfs/pull/1955: progress meter bugfixes

**Comment by celsodantas** [comment]
\o/

**Pull request merged**